ansible/roles_studentvm/studentvm_ocp4/defaults/main.yml

```diff
@@ -16,6 +16,8 @@ studentvm_ocp4_helm_version: '3.1.1'
 studentvm_ocp4_tkn_version: '0.8.0'
 # https://github.com/knative/client/releases
 studentvm_ocp4_kn_version: 'v0.12.0'
+# https://github.com/istio/istio/releases
+studentvm_ocp4_istioctl_version: '1.5.2'
```

Comment:
I had left that out on purpose. According to istio Chat istioctl is not a part of OpenShift.... But honestly don't feel strongly about it.

Reply:
Mmmm, I would prefer we add it. Tekton, knative, etc. are not a "core" part of the platform either since you have to install them. OSSM comes with OpenShift, so that is close enough for me.

Plus it would be silly to have a role just to install that one package.